### PR TITLE
Create vansjyo.c

### PR DESCRIPTION
**Pull request by vansjyo**
This Pull Request resolves Issue #13

### CODE REVIEW

File: Insertion_Sort/vansjyo.c [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+{
+   int i;
+   for (i=0; i < n; i++)
+       printf("%d ", arr[i]);
+   printf("\n");
+}
+
+int main()
+{
+    int arr[] = {12, 11, 13, 5, 6};
```
Comment (MJ10) [inline Insertion_Sort/vansjyo.c]: Accept the array from user input instead of hard coding the values

**Comment by MJ10** [comment]
@vansjyo Are there any updates or should we close this and mark it as spam?

**Comment by vansjyo** [comment]
Il do it.

**Comment by vansjyo** [comment]
Close it..

**Pull request closed without merging**